c7n/offhours.py

```diff
@@ -301,10 +299,8 @@ def get_local_tz(self, parts):
 
 
     def is_custom(self, parts):
-        if len(parts) == 3 and parts.get("on") and parts.get("off"):
-            return True
-        else:
-            return False
+        return "tz" in parts and "on" in parts and "off" in parts
```

Comment:
Just felt this was cleaner way to check for the basic input items for custom logic.